components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -843,132 +841,62 @@ def read_and_update_settings_json(settings_file_path: pathlib.Path, updates: Dic
-        get_clp_home()
-        / "var"
-        / "www"
-        / "log-viewer-webui"
-        / "server"
-        / "dist"
-        / "server"
-        / "settings.json"
+    client_settings_json = read_and_update_settings_json(
+        client_settings_json_path, client_settings_json_updates
     )
+    with open(client_settings_json_path, "w") as client_settings_json_file:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)